src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
 </Row>
 
+#### Using the grid
+
+By default the container’s width is set to the size of the text label with 64px fixed padding on the right side and 16px fixed padding on the left. However, when possible set the button container’s relative position to the responsive layout grid and match buttton width to the other elements on the page. Ideally, when using groups of related buttons (not including ghost buttons), they should all be the same width.
+See [button groups](#Button-groups) below for more detailed information.
+
 <Row>
 <Column colLg={8}>
 
-![Secondary / Primary button alignment in forms](images/button-usage-7.png)
+![Example of a primary button matching the width of a card in a layout](images/button_usage_15_test.png)
 
-<Caption>Secondary / Primary button alignment in forms</Caption>
+<Caption>
+  Example of a primary button matching the width of a card in a layout
+</Caption>
 
 </Column>
 </Row>
 
-### Multistep forms
+### Button groups
 
-When the primary action implies a navigation step forward, as in a wizard, align single buttons or Secondary/Primary button groups to the right. This position conveys the “next step” intention.
+Button groups are a useful way of aligning buttons that have a relationship. Group the buttons logically into sets based on usage and importance. Too many calls to action will overwhelm and confuse users so they should be avoided.
+
+As mentioned in the [Emphasis section](#Emphasis), you don’t necessarily need to use the buttons in the order that their labels imply. Either a secondary or a tertiary button can be used in conjunction with a primary button. In fact, due to the visual weight of the secondary button, it’s recommended to use tertiary or ghost buttons in layouts with more than three calls to action.
 
 <Row>
 <Column colLg={8}>
 
-![Primary button alignment in a wizard](images/button-usage-8.png)
+![multiple buttons in a layout](images/button_usage_16.png)
 
-<Caption>Single button alignment in a wizard</Caption>
+</Column>
+</Row>
+
+<DoDontRow>
+  <DoDont caption="Do use a primary and two of the same lower emphasis buttons in a button group.">
+
+![Do use a primary and two of the same lower emphasis buttons in a button group.](images/button_usage_17.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not mix primary, secondary, and tertiary buttons in the same button group.">
+
+![Do not mix primary, secondary, and tertiary buttons in the same button group.](images/button_usage_18.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Button groups and the grid
+
+Ideally, when using groups of related buttons (not including ghost buttons), they should all be the same width. This can be achieved in one of two ways, both of which are acceptable. The first approach involves using the [narrow grid mode](/guidelines/2x-grid/code). In this situation each button would be set individually on the column grid.
+
+<Row>
+<Column colLg={8}>
+
+![Example of hanging buttons on the grid](images/button_usage_19.png)
```

Comment:
Might help to defer on this example until we have docs and implementation for hanging on the grid. This can cause a mismatch between design intent and development where a developer is unsure how to implement something and leads to overrides in our components.

Reply:
Here I'm very concerned about the source of this problem, which is our narrow-grid mode guidance... that guidance is up in the 2x-grid and we show these hanging button examples. In fact, this paragraph links to it. Buttons are seen with this left-side hang all over product and Gatsby... people especially do it in dashboards to align the button with the tiles. 

I changed the copy to this:

Ideally, when using groups of related buttons (not including ghost buttons), they should all be the same width. This can be achieved in one of two ways, both of which are acceptable. The first approach involves using the [narrow grid mode](/guidelines/2x-grid/code). In this situation each button would be set individually on the column grid. Note: Carbon developers are working on the narrow grid mode, currently left-hanging buttons can only be achieved with an override.